crossfire/clients/occurrences.py

```diff
@@ -181,11 +181,26 @@ def __call__(self):
         return self.data
 
 
-def _flatten_df(row, column_name):
-    column_data = row[column_name]
-    return Series(
-        {f"{column_name}_{key}": value for key, value in column_data.items()}
-    )
+def _flatten_df(data, nested_columns):
+    def _flatten_col(row, column_name):
+        column_data = row[column_name]
+        return Series(
+            {
+                f"{column_name}_{key}": value
+                for key, value in column_data.items()
+            }
+        )
+
+    keys = set(data.columns) & nested_columns
+    for key in keys:
+        data = concat(
+            [
+                data.drop(key, axis=1),
+                data.apply(_flatten_col, args=(key,), axis=1),
+            ],
+            axis=1,
+        )
+    return data
```

Comment:
Commit created to addres [your comment](https://github.com/FelipeSBarros/crossfire/pull/103#discussion_r1511572073)..